Tasks/NugetPublisher/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -27,7 +27,7 @@
   "loc.messages.LegacyNuGetHint": "If %s is older than NuGet 3.2, you may need to edit the build definition and check the 'Path to NuGet.exe is below version 3.2' checkbox",
   "loc.messages.BuildIdentityPermissionsHint": "For internal feeds, make sure the build service identity '%s' [%s] has access to the feed.",
   "loc.messages.NoCredProviderOnAgent": "CredentialProvider.TeamBuild.exe not found on the agent. Falling back to config-based credential flow.",
-  "loc.messages.NotARegularFile": "%s is not a solution file. Check the 'path to solution or packages.config' property of the NuGetInstaller task.",
+  "loc.messages.NotARegularFile": "%s is not a valid path. Check the 'Path/Pattern to nupkg' property of the NuGetPublisher task.",
```

Comment:
It is a valid path, it's just not a regular file. (most likely it's a directory, but especially on non-windows it could be a device file or something)